.github/workflows/conformance-aws-cni.yaml

```diff
@@ -56,7 +56,7 @@ concurrency:
     ${{
       (github.event_name == 'schedule' && github.sha) ||
       (github.event_name == 'issue_comment' && (
-        github.event.comment.body == '/ci-awscni') ||
+        github.event.comment.body == '/ci-awscni' ||
```

Comment:
And to think I did a review specifically looking at unmatched parenthesis :facepalm: